### PR TITLE
fix(medusa): Issue when ordering with multiple columns

### PR DESCRIPTION
**Pull request by adrien2p**
**What**
No true fix due to the same issue as [here](https://github.com/typeorm/typeorm/issues/6294) but at least the pagination works again. The ordering can't be applied on multiple columns/relation as it produce the wrong SQL.

FIXES CORE-1193

### CODE REVIEW

**Comment by adrien2p** [comment]
@pKorsholm it should be good now. Can I get you to test in your branch? 
we have to keep in mind that typeorm is bugged when it comes to ordering by multiple columns and relations ...

**Comment by adrien2p** [comment]
@olivermrbl do you thinkn we can merge that one ?

**Review by olivermrbl — APPROVED** [review]
LGTM!

File: packages/medusa/src/repositories/order.ts [hunk 1]
```diff
@@ -34,6 +34,8 @@ export const OrderRepository = dataSource.getRepository(Order).extend({
           relations: rels,
           withDeleted:
             topLevel === ITEMS_REL_NAME || topLevel === REGION_REL_NAME,
+          // TODO: Remove eager relations and then pass to `query` if needed
```
[the remark this answers is not in the document]
Comment (adrien2p) [reply packages/medusa/src/repositories/order.ts:37]: I ll remove it entirely as it was part of a test I was doing, will discuss that part later if that'scool with you

**Review by pKorsholm — APPROVED** [review]
LGTM!

**Comment by olivermrbl** [comment]
@adrien2p Can I get you to investigate why tests are not passing?

**Comment by adrien2p** [comment]
> @adrien2p Can I get you to investigate why tests are not passing?

I am on it 💪

**Review by srindom — APPROVED** [review]
LGTM!

**Comment by minikdev** [comment]
is this fixed? I want to order by an added select field based on a joined entity and it does not get applied.

**Pull request merged**